static/js/components/CourseList.js

```diff
@@ -1,22 +1,32 @@
 import React from 'react';
 import _ from 'lodash';
-
+import Button from 'react-bootstrap/lib/Button';
 import {
   makeCourseStatusDisplay,
   makeCourseProgressDisplay,
 } from '../util/util';
+import {
+  STATUS_PASSED,
+} from '../constants';
 
 class CourseList extends React.Component {
   render() {
     const { dashboard } = this.props;
+    var coursesPassed = 0;
+    var totalCourses = 0;
```

Comment:
can you use `let` instead of `var` here? ES2015 introduced the `let` and `const` keywords for variable binding, they have better behavior variable scoping and so on: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/let